src/NextjsBuild.ts

```diff
@@ -101,7 +110,7 @@ export class NextjsBuild extends Construct {
 
   private build() {
     const buildPath = this.props.buildPath ?? this.props.nextjsPath;
-    const buildCommand = this.props.buildCommand ?? 'npx open-next@2 build';
+    const buildCommand = this.props.buildCommand ?? 'npx open-next@2.2 build';
```

Comment:
@bestickley FYI I bumped this to 2.2 because some of the features of revalidation aren't available in 2.0/2.1 of `open-next`

Reply:
@kevin-mitchell, thank you for this. I've noticed this too, that `npx open-next@2 build` doesn't use the latest version of 2.X for `open-next`. Do you know why this is? I thought specifying `@2` would do latest major version of 2, right?

Reply:
I believe (though it's not 100% clear to me if this is the correct behavior) `@2` will use any version of 2 in `npx` cache (`~/.npm/_npx/` on Mac). I haven't spent a ton of time testing so I'm not sure if being more explicit (e.g. `@^2` or perhaps `@~2.2`) would do the trick, or if this just works for me just now because it wasn't in the NPX cache and https://github.com/npm/cli/issues/6804 is a legit issue.

Reply:
@kevin-mitchell, could you change this to `npx open-next@^2 build` so we don't have to keep updating this? Make sure to update comment to.

Reply:
yep, done!